uk.co.compucorp.civicrm.hrcore/js/src/hrcore/helpers.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env amd */
+
+(function (CRM, $) {
+  define(function () {
+    'use strict';
+
+    return {
+      appendHelpIcon: appendHelpIcon,
+      getCiviCRMFormLabel: getCiviCRMFormLabel
+    };
+
+    /**
+     * Appends a help icon to a given element
+     *
+     * @param {jQuery} $element
+     * @param {String} title the title of the help icon
+     * @param {String} helpId the ID of the helper
+     * @param {String} helpFile the path to the helper template
+     */
+    function appendHelpIcon ($element, title, helpId, helpFile) {
+      var $helpIcon = $('<a href class="helpicon" title="' + title + '"></a>');
+
+      $element.append($helpIcon);
+
+      $helpIcon.on('click', function () {
+        CRM.help('', { id: helpId, file: helpFile });
+
+        return false;
```

Comment:
why is this necessary? The `href` property is empty, so the click event shouldn't do anything?

Reply:
i'm not questioning why `href` is there, i'm asking why do you need to return `false`

Reply:
Apologies. `return false` is needed because if you specify `href` as an empty attribute it will simply refresh the current URL if you don't return `false`.